invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -145,6 +146,9 @@
                   <div id="recordManagementMobile"
                        role="dialog"
                        class="ui flowing popup transition hidden"
+                       data-record='{{ record | tojson }}'
+                       data-permissions='{{ permissions | tojson }}'
+                       data-is-draft="{{ is_draft | tojson }}"
```

Comment:
I don't think this is needed here, as the mobile version uses the same data as for `#recordManagement` in `landing_page/index.js` ? Ref https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js#L29

Reply:
yes, you are right and I saw it but I prefer to be explicit in the code, in the previous implementation this was a bit "hidden"